src/pages/DoctorDashboard.tsx

```diff
@@ -9,12 +10,156 @@ import {
     Navigate,
     Outlet,*/
 } from "react-router-dom";
+import {Vaccination, Visit, AuthContextType} from "../components/types";
+import {addDays, getBeginningOfWeek} from "../components/dateUtils";
+import DatePicker from 'react-date-picker';
+import { getSlots } from '../logic/doctorAPI';
+import { useAuth } from '../App';
+
 function DoctorDashboard() {
     const navigate = useNavigate()
+    let today = getBeginningOfWeek(new Date());
+    today.setHours(0,0,0,0);
+    const [startDate, onStartDateChange] = useState<Date>(today);
+    const [endDate, onEndDateChange] = useState<Date>(addDays(today, 6));
+    const [reserved, setReserved] = useState<string>('-1');
+    const [page, setPage] = useState<number>(1);
+    const [maxPage, setMaxPage] = useState<number>(10);
+    const radios = [
+        { name: 'All', value: '-1' },
+        { name: 'Free', value: '0' },
+        { name: 'Reserved', value: '1' },
+    ];
+    let auth: AuthContextType = useAuth();
+    const sampleInfo: Visit[] = [
```

Comment:
Ja bym to wywalił poza komponent. Tuż po importach. Tak żeby niepotrzebne nie zaśmiecać samego komponentu.

Reply:
Will do